src/citrine/resources/project.py

```diff
@@ -157,6 +157,22 @@ def share(self,
             "resource": {"type": resource_type, "id": resource_id}
         })
 
+    def make_public(self,
+                    resource_type: str,
```

Comment:
Are the `resource_type`s documented?  If we're expecting the user to provide the resource type, then they should know what to provide.